docs/source/docs/installation/wiring.md

```diff
@@ -14,19 +14,19 @@ Please make sure your chosen power supply can provide enough power for your copr
 We recommend users use the [SnakeEyes Pi Hat](https://www.playingwithfusion.com/productview.php?pdid=133) as it provides passive power over ethernet (POE) and other useful features to simplify wiring and make your life easier.
 :::
 
-### Recommended: Coprocessor with Passive POE (Gloworm, Pi with SnakeEyes, Limelight)
+### Recommended: Coprocessor with Passive POE (Pi with SnakeEyes and Limelight)
 
 1. Plug the [passive POE injector](https://www.revrobotics.com/rev-11-1210/) into the coprocessor and wire it to PDP/PDH (NOT the VRM).
 2. Add a breaker to relevant slot in your PDP/PDH
-3. Run an ethernet cable from the passive POE injector to your network switch / radio (we *STRONGLY* recommend the usage of a network switch, see the [networking](networking.md) section for more info.)
+3. Run an ethernet cable from the passive POE injector to your network switch / radio (we _STRONGLY_ recommend the usage of a network switch, see the [networking](networking.md) section for more info.)
```

Comment:
I believe this recommendation only applies. If you have the old open mesh radio, the new one should not need a switch.

Reply:
Unless you use 12 coprocessors 😁